### PR TITLE
added note about PKU2U

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5199

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing and style checks needed for this PR. Thanks.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -82,7 +82,8 @@ In organizations using only Azure AD, you can connect from an Azure AD-joined PC
 - Password
 - Windows Hello for Business, with or without an MDM subscription. 
 
-
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
> [!NOTE]
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 2]
```diff
@@ -82,7 +82,8 @@ In organizations using only Azure AD, you can connect from an Azure AD-joined PC
...
+>To be able to connect to Azure Active Directory-joined computers, RDP client must [allow Public Key Cryptography Based User-to-User (PKU2U) authentication requests to use online identities](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities).
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
> To be able to connect to Azure Active Directory-joined PCs, the RDP client must [allow Public Key Cryptography Based User-to-User (PKU2U) authentication requests to use online identities](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities).
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 3]
```diff
@@ -82,7 +82,8 @@ In organizations using only Azure AD, you can connect from an Azure AD-joined PC
 - Password
 - Windows Hello for Business, with or without an MDM subscription. 
 
-
+> [!NOTE]
+> If RDP client running Windwos Server 2016 or Windows Server 2019, to be able to connect to Azure Active Directory-joined PCs, it must [allow Public Key Cryptography Based User-to-User (PKU2U) authentication requests to use online identities](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities).
```
Comment (illfated) [inline windows/client-management/connect-to-remote-aadj-pc.md]: typo + semantics :
```suggestion
> If the RDP client is running Windows Server 2016 or Windows Server 2019, to be able to connect to Azure Active Directory-joined PCs, it must [allow Public Key Cryptography Based User-to-User (PKU2U) authentication requests to use online identities](https://docs.microsoft.com/windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities).
```

Comment (JohanFreelancer9) [reply windows/client-management/connect-to-remote-aadj-pc.md]: Thanks illfated. Missed that, must have been early morning hours.

**Pull request merged**